lib/presentation/atoms/text_atom.dart

```diff
@@ -58,5 +63,5 @@ enum TextVariant {
   regular,
   medium,
   title,
-  ;
+  time;
```

Comment:
use titleLarge or headlineMedium, there is no reason to add a new one

Reply:
Changed to TextVariant.title

Reply:
Thanks